app/templates/layouts/_base.html

```diff
@@ -43,15 +43,37 @@
         <a class="skip__link" href="#main">Skip to content</a>
       </div>
 
-      {% block top_bar %}{% endblock %}
+      <div class="page__header">{% block header %}{% endblock %}</div>
+      {%- set show_previous = previous_location != None and previous_location|length != 0 -%}
```

Comment:
This flag seems redundant, as previous_location isn't set. We should only need to do:

     {% if previous_location %}
       <div class="container">
         <a class="page__previous" id="top-previous" href="{{previous_location}}">Previous</a>
       </div>
     {% endif %}